src/lib.rs

```diff
@@ -167,17 +177,22 @@
 //!
 //!
 
+#[allow(unused_imports)]
+#[macro_use]
+extern crate specs_derive;
```

Comment:
Just discussed with @kvark, we decided not to reexport it here so it doesn't get included if it's unnecessary for the user.